pkg/executables/kubectl.go

```diff
@@ -1222,3 +1222,27 @@ func (k *Kubectl) GetDaemonSet(ctx context.Context, name, namespace, kubeconfig
 
 	return obj, nil
 }
+
+func (k *Kubectl) GetPackagesFromKubectl(ctx context.Context, packageInstanceName, kubeconfig string, output string) (string, error) {
+	params := []string{
+		"get", "packages", "--kubeconfig", kubeconfig, "--namespace", constants.EksaPackagesName,
```

Comment:
```suggestion
		"get", "packages",  "--namespace", constants.EksaPackagesName,
```

Reply:
I tested this and it looks like kubeconfig is required for the command to work.

Reply:
Not if you set the KUBECONFIF env var or use the default location